### PR TITLE
Doc cleanup

### PR DESCRIPTION
**Pull request by vladimir-ch**

### CODE REVIEW

**Comment by kortschak** [comment]
LGTM, but wait for @btracey since I obviously can't read today.

File: cgo/lapack.go [hunk 1]
```diff
@@ -737,19 +736,19 @@ func (impl Implementation) Dorgbr(vect lapack.DecompUpdate, m, n, k int, a []flo
 
 // Dorglq generates an m×n matrix Q with orthonormal rows defined by the
 // product of elementary reflectors as computed by Dgelqf.
-//  Q = H(0) * H(2) * ... * H(k-1)
-// Dorglq is the blocked version of dorgl2 that makes greater use of level-3 BLAS
+//  Q = H_k ... H_2 * H_1
```
Comment (btracey) [inline cgo/lapack.go]: I think this should be zero indexed. I tried to be consistent with that in the documentation. It doesn't matter for the documentation of this function, but it does for others.

File: cgo/lapack.go [hunk 2]
```diff
@@ -779,16 +778,16 @@ func (impl Implementation) Dorglq(m, n, k int, a []float64, lda int, tau, work [
 
 // Dorgqr generates an m×n matrix Q with orthonormal columns defined by the
 // product of elementary reflectors as computed by Dgeqrf.
-//  Q = H(0) * H(2) * ... * H(k-1)
-// Dorgqr is the blocked version of dorg2r that makes greater use of level-3 BLAS
+//  Q = H_1 * H_2 ... H_k
```
Comment (btracey) [inline cgo/lapack.go]: Same here.

File: lapack64/lapack64.go [hunk 3]
```diff
@@ -52,7 +52,7 @@ func Potrf(a blas64.Symmetric) (t blas64.Triangular, ok bool) {
 // given the LU decomposition of the matrix. The condition number computed may
 // be based on the 1-norm or the ∞-norm.
 //
-// The slice a contains the result of the LU decomposition of A as computed by Dgetrf.
+// The matrix a contains the result of the LU decomposition of A as computed by Getrf.
```
Comment (btracey) [inline lapack64/lapack64.go]: How about just "a contains"

File: lapack64/lapack64.go [hunk 4]
```diff
@@ -115,27 +115,27 @@ func Gels(trans blas.Transpose, a blas64.General, b blas64.General, work []float
...
-// The lower triangle of a contains the matrix L. The lower triangular elements
-// (not including the diagonal) contain the elementary reflectors. Tau is modified
-// to contain the reflector scales. Tau must have length at least min(m,n), and
-// this function will panic otherwise.
+// Gelqf computes the LQ factorization of the m×n matrix A using a blocked
+// algorithm. A is modified to contain the information to construct L and Q. The
+// lower triangle of a contains the matrix L. The elements above the diagonal
+// and the slice tau represent the matrix Q. tau is modified to contain the
+// reflector scales. tau must have length at least min(m,n), and this function
+// will panic otherwise.
```
Comment (btracey) [inline lapack64/lapack64.go]: Okay, I guess I wasn't consistent. Could you make this zero-indexed as well

**Comment by btracey** [comment]
LGTM with the minor fixes.

File: cgo/lapack.go (hunk 1 above)
Comment (vladimir-ch) [inline cgo/lapack.go]: I made H zero-indexed everywhere in cgo and lapack docs (not in native). But if I were to be rigorous, there are still some related inconsistencies and missing explanations ...
Anyway, done.

File: cgo/lapack.go (hunk 2 above)
Comment (vladimir-ch) [inline cgo/lapack.go]: Done.

File: lapack64/lapack64.go (hunk 3 above)
Comment (vladimir-ch) [inline lapack64/lapack64.go]: Done.

File: lapack64/lapack64.go (hunk 4 above)
Comment (vladimir-ch) [inline lapack64/lapack64.go]: Done.

**Pull request merged**